### PR TITLE
fix(FEC-11199): NaN shown instead of duration for bumper videos

### PR DESCRIPTION
**Pull request by yairans**
### Description of the Changes

Ignore the TIME_UPDATE before the bumper actually loaded 

Solves FEC-11199

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

**Review by Yuvalke — APPROVED** [review]

File: src/bumper.js [hunk 1]
```diff
@@ -387,13 +387,14 @@ class Bumper extends BasePlugin implements IMiddlewareProvider, IAdsControllerPr
   }
 
   _onTimeUpdate(): void {
-    this._adBreak &&
+    if (this._adBreak && this._bumperState !== BumperState.LOADING) {
```
Comment (Yuvalke) [inline src/bumper.js:390]: change `_bumperState` to `state`

**Review by Yuvalke — CHANGES_REQUESTED** [review]

File: src/bumper.js (hunk 1 above)
Comment (yairans) [reply src/bumper.js:390]: why it metter?

Comment (Yuvalke) [reply src/bumper.js:390]: it's not, just keep the same naming

Comment (Yuvalke) [reply src/bumper.js:390]: Isn't it better `state === playing`? seems like `done`/`idle`/`loaded`/`loading` all shouldn't dispatch ad_progress, right?

Comment (yairans) [reply src/bumper.js:390]: could be LOADED as well

Comment (yairans) [reply src/bumper.js:390]: what do you mean same naming? `_bumperState` it's the private and `state` it's the public getter

**Review by Yuvalke — APPROVED** [review]

**Pull request merged**